rules/module_extensions.bzl

```diff
@@ -0,0 +1,44 @@
+"""Definitions for bzlmod module extensions."""
+
+load(
+    "//rules:repositories.bzl",
+    "rules_ios_dependencies",
+    "rules_ios_dev_dependencies",
+)
+load(
+    "//tools/toolchains/xcode_configure:xcode_configure.bzl",
+    _xcode_configure = "xcode_configure",
+)
+
+def _non_module_deps_impl(_):
+    rules_ios_dependencies(
+        load_xchammer_dependencies = True,
+        load_bzlmod_dependencies = False,
+    )
+
+non_module_deps = module_extension(implementation = _non_module_deps_impl)
+
+def _non_module_dev_deps_impl(_):
+    rules_ios_dev_dependencies(load_bzlmod_dependencies = False)
+
+non_module_dev_deps = module_extension(implementation = _non_module_dev_deps_impl)
```

Comment:
This loads and runs the xcode configuration that was previously in the WORKSPACE

Reply:
Isn't it fair to say that bzlmod was updated to work with existing repository rules? e.g. the [`use_repo`](https://github.com/bazel-ios/rules_ios/pull/703/files#diff-6136fc12446089c3db7360e923203dd114b6a1466252e71667c6791c20fe6bdcR58) wrapper can help use repos?